scripts/utils/changeNumber.js

```diff
@@ -3,7 +3,7 @@ export default (number, type)  => {
 
   if(number != null){
     if(number > 0) {
-      if(type === 'top') {
+      if(type === 'percentage') {
```

Comment:
Since type can only be undefined or 'percentage', can we replace this with an isPercentage boolean?